### PR TITLE
Feat: Router for navigation and API calls to fetch data are added in Quiz App

### PR DESCRIPTION
**Pull request by sakleprajwal**
Routers to navigate and API calls to fetch data are added in Quiz App

https://user-images.githubusercontent.com/40752468/160841352-44cbe6ae-8f10-431d-9018-99a7028c69a6.mp4

### CODE REVIEW

**Review by Gautam2305 — APPROVED** [review]
everything looks good.

API calls to fetch data are looking good. Liked how you used map & useEffect too.
Overall looks good. Go for merge after receiving another review.

File: src/pages/Question/Question.js [hunk 1]
```diff
@@ -1,37 +1,67 @@
...
+import { v4 as uuid } from "uuid";
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
 import "../../styles/base.css"
 
 const Question = () => {
+    const [questionObjs, setQuestionObjs] = useState([])
+
+    const fetchQuestions = async () => {
+        const { data } = await axios.get("https://opentdb.com/api.php?amount=5&category=17&difficulty=easy&type=multiple");
```
Comment (therajatg) [inline src/pages/Question/Question.js:11]: can have try-catch block

**Review by therajatg — APPROVED** [review]
else all looks good

File: src/pages/Question/Question.js (hunk 1 above)
Comment (sakleprajwal) [reply src/pages/Question/Question.js:11]: Thanks for highlighting

**Pull request merged**